queries/rescript/highlights.scm

```diff
@@ -0,0 +1,340 @@
+(comment) @comment @spell
+
+; Identifiers
+;------------
+; Escaped identifiers like \"+."
+((value_identifier) @constant.macro
+  (#match? @constant.macro "^\\.*$"))
```

Comment:
Can this be replaced with a Lua pattern?

Reply:
(I think it can, so it should.)

Reply:
Yes I agree, if I resolved this without fixing it was by accident haha. Apologies